### PR TITLE
Add 'ascending', 'inplace', na_position` option to sort_values with doctest/docstring

### LINKED ISSUE #132: Support ascending in sort_values
I want to sort values in descending order using sort_values, and although sort_values is supported, the ascending keyword is not.
Screen Shot 2019-04-22 at 8 06 20 AM

### PR DESCRIPTION
**Pull request by HyukjinKwon**
This PR adds 'ascending' option to sort_values with doctest/docstring.

Resolves #132
Closes #53

### CODE REVIEW

**Comment by HyukjinKwon** [comment]
cc @rxin

File: databricks/koalas/frame.py [hunk 1]
```diff
@@ -661,8 +661,74 @@ def get(self, key, default=None):
         except (KeyError, ValueError, IndexError):
             return default
 
-    def sort_values(self, by):
-        return DataFrame(self._sdf.sort(by), self._metadata.copy())
+    def sort_values(self, by, ascending=True):
```
Comment (rxin) [inline databricks/koalas/frame.py]: ascending is Pandas is pretty sophisticated. You can also specify for each of the sort columns whether it is asc or desc. 

https://github.com/databricks/koalas/pull/53

Maybe you should take over this PR. It implements ascending and also na_position, inplace.

Comment (HyukjinKwon) [reply databricks/koalas/frame.py]: Yup, will take that over.

**Comment by HyukjinKwon** [comment]
I am getting this in. All comments in here and #53 are addressed, doc is inlined too, and the tests passed.

**Comment by rxin** [comment]
Cool thanks!

**Comment by HyukjinKwon** [comment]
Hm? wait. It passed in my local but failed in Travis. Let me check.

**Comment by HyukjinKwon** [comment]
Merged. Thanks for review @rxin.

**Pull request merged**